.github/workflows/ruby.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   test:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Focal Fossas approve this change 😸 
![image](https://user-images.githubusercontent.com/22104233/151821990-81697847-a620-485b-813f-9adf59da557a.png)